Milyli.ScriptRunner.Core/Services/RelativityScriptRunner.cs

```diff
@@ -37,73 +37,75 @@ public class RelativityScriptRunner : IRelativityScriptRunner
-			foreach (var schedule in schedules)
-			{
-				if (schedule.DirectSql)
-				{
-					this.ExecuteDirectSqlJob(schedule);
-				}
-				else
-				{
-					this.ExecuteScriptJob(schedule);
-				}
+			foreach (var schedule in schedules)
+			{
+				if (schedule.DirectSql)
+				{
+					this.ExecuteDirectSqlJob(schedule);
+				}
+				else
+				{
+					this.ExecuteScriptJob(schedule);
+				}
 			}
-		}
-
-		/// <inheritdoc />
-		public void ExecuteDirectSqlJob(JobSchedule job)
-		{
-			var workspace = new RelativityWorkspace { WorkspaceId = job.WorkspaceId };
-			var searchIds = new List<int>();
-			var activationStatus = this.jobScheduleRepository.StartJob(job);
-
+		}
+
+		/// <inheritdoc />
+		public void ExecuteDirectSqlJob(JobSchedule job)
+		{
+			var workspace = new RelativityWorkspace { WorkspaceId = job.WorkspaceId };
+			var searchIds = new List<int>();
+			var activationStatus = this.jobScheduleRepository.StartJob(job);
+
 			if (activationStatus == JobActivationStatus.Started)
-			{
-				// Not a fan of the double try-catch, but the outer clause simply replicates the existing
-				// pattern of job execution in ExecuteScriptJob. If agent execution is refactored in the future the outer clause should be removed.
+			{
+				// Not a fan of the double try-catch, but the outer clause simply replicates the existing
+				// pattern of job execution in ExecuteScriptJob. If agent execution is refactored in the future the outer clause should be removed.
 				try
-				{
-					var inputs = this.jobScheduleRepository.GetJobInputs(job);
-					var script = RelativityHelper.InWorkspace((client, _) => client.Repositories.RelativityScript.ReadSingle(job.RelativityScriptId), workspace, this.relativityClient.GetRelativityClient());
-					var originalInputs = RelativityHelper.InWorkspace((client, _) => client.Repositories.RelativityScript.GetRelativityScriptInputs(script), workspace, this.relativityClient.GetRelativityClient());
-					var scriptSql = script.Body.AsXmlDocument.GetElementsByTagName("action").Item(0).InnerText;
-					scriptSql = this.relativityScriptProcessor.SubstituteGlobalVariables(workspace.WorkspaceId, scriptSql);
-					scriptSql = this.relativityScriptProcessor.SubstituteScriptInputs(inputs, originalInputs, scriptSql);
-					searchIds = this.relativityScriptProcessor.GetSavedSearchIds(inputs, originalInputs).ToList();
-
-					var searchTables = this.searchTableManager.CreateTablesAsync(workspace.WorkspaceId, searchIds, job.Id, job.MaximumRuntime).GetAwaiter().GetResult();
-					scriptSql = this.relativityScriptProcessor.SubstituteSavedSearchTables(inputs, originalInputs, searchTables, scriptSql);
-
-					try
-					{
-						this.helper.GetDBContext(workspace.WorkspaceId).ExecuteNonQuerySQLStatement(scriptSql, job.MaximumRuntime);
-					}
-					finally
-					{
-						this.searchTableManager.DeleteTables(workspace.WorkspaceId, searchTables.Values);
-					}
-				}
+				{
+					Logger.Info($"Executing script {job.Name} via direct SQL.");
```

Comment:
The default generate name is of the format {Workspace Name - Script Name} so this provides plenty of information. If someone has given a job a custom name I would hope the name they chose makes sense to them.